hub/management/commands/import_council_emission_cluster.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import add_gss_codes, filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+shaders = [
+    {"title": "Agriculture", "shader": "green"},
+    {"title": "City of London", "shader": "lightgrey"},
+    {"title": "Industry/Commerical/Domestic", "shader": "darkblue"},
+    {"title": "Public sector", "shader": "lightblue"},
+    {"title": "Transport/Domestic", "shader": "lightpurple"},
+    {"title": "Urban Mainstream", "shader": "darkgrey"},
+]
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council emissions data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Authority Emissions Cluster",
```

Comment:
I’d probably word this as “Emissions profile” … not even sure it needs “local authority” in it, given it applies to the whole area, not just the council itself.